src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,36 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    static class Machine {
+        public void doWork() {
+            System.out.println("doWork");
+        }
+        public void stopWork() {
+            System.out.println("stopWork");
+        }
+    }
+    static class Truck extends Machine {
+        public void doWork() {
+            super.doWork();
+        }
```

Comment:
The `doWork` method in `Truck` class only calls `super.doWork()`. If the task requires specific behavior for each subclass, consider adding unique functionality here.